src/main/java/step2/StringOperation.java

```diff
@@ -0,0 +1,62 @@
+package step2;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.DoubleToIntFunction;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public final class StringOperation {
+
+    private StringOperation() {}
+    private static final class SingletonHelper {
+        public static final StringOperation INSTANCE = new StringOperation();
+    }
+    public static StringOperation getInstance(){
+        return SingletonHelper.INSTANCE;
+    }
+
+    private int calculateFourOperation(int a, int b, char operator){
+        switch (operator){
+            case '+': return add(a, b);
+            case '-': return subtract(a, b);
+            case '*': return multiply(a, b);
+            case '/': return divide(a, b);
+            default:
+                throw new IllegalArgumentException();
+        }
+    }
+
+    public int calculate(String input){
+        if(input == null || input.equals("")) throw new IllegalArgumentException();
```

Comment:
검증 기능을 추가해주셨군요 :) 💯 , 
이 부분에 대해서는 `public void validateEmpty()` 라는 메서드로 분리해보는 것은 어떨까요?  
그러면 `calculate()`라는 메서드 안에 `검증`이랑 `계산`이라는 2가지 기능이 분리 될 것이고   
위와 같이 빈 값이 입력되는 로직이 추가로 필요하다면 `validateEmpty()`를 호출하여 재사용할 수 있을 것 같네요!

Reply:
```suggestion
        if(Objects.isNull(input) || String.isEmpty(input)) {
            throw new IllegalArgumentException();
        }
```
한 줄 코드라도 `{}`를 붙여주셔야 안정성이 높아지니 참고 부탁드리고요  
위와 같이 라이브러리를 이용하는 것도 좋을 것 같네요!!